File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        if(nums.size() == 0) return 0;
        int unique = 1;
        for(int i=1; i<nums.size(); i++){
            if(nums[i] != nums[i - 1]){
                nums[unique] = nums[i];
                unique++;
            }
        }
        return unique;
    }
};

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function(nums) {
    if(nums.length == 0) return 0;

    var unique = 1;
    for(var i=1; i<nums.length; i++){
        if(nums[i] != nums[i-1]){
            nums[unique] = nums[i];
            unique++;
        }
    }
    return unique;
};